### PR TITLE
replace hardcoded maildomain with variable

### PR DESCRIPTION
**Pull request by stoeps13**
Maildomain for demo LDAP Users was hardcoded in the jinga2 template, replaced it with the already defined variable.

Additionally replaced the admin user variables like it was intentionally thought (or better I think it was). The variables in inventory let you define ldap_admin userid and password, but it is not used in the template.

### CODE REVIEW

File: roles/third_party/ldap-install/templates/users.ldif.j2 [hunk 1]
```diff
@@ -4,14 +4,14 @@ objectClass: top
 dc: {{ __ldap_tld }}
 
 
-dn: uid={{ __ldap_userid }}1,{{ __ldap_olc_suffix }}
+dn: uid={{ __ldap_admin_user }}1,{{ __ldap_olc_suffix }}
```
Comment (dejo1307) [inline roles/third_party/ldap-install/templates/users.ldif.j2:7]: I'm probably missing the context here, but __ldap_admin_user is, by default, Admin, the user which has in context admin rights on LDAP. 

__ldap_userid, in another hand, is some user ID you can give it to create a bunch of users where the first of them will become CNX admin as well, so those are two different domains. I'm not sure I would change this.

File: roles/third_party/ldap-install/templates/users.ldif.j2 [hunk 2]
```diff
@@ -52,7 +52,7 @@ objectClass: top
 cn: Joe Jones{{ counter }}
 sn: Jones{{ counter }}
 givenName: Joe
-mail: {{ __ldap_userid }}{{ counter }}@connections.example.com
+mail: {{ __ldap_userid }}{{ counter }}@{{ __ldap_user_mail_domain }}
```
Comment (dejo1307) [inline roles/third_party/ldap-install/templates/users.ldif.j2:55]: This definitely makes sense, not sure why I didn't see it hardcoded.

**Review by dejo1307 — CHANGES_REQUESTED** [review]

**Comment by stoeps13** [comment]
Was irritated from ldap_user_admin_password, now it should be OK again

**Review by stoeps13 — COMMENTED** [review]
Removed the admin user, now it should be fixed, but the original ldap_user_admin_password was wrong here too.

File: roles/third_party/ldap-install/templates/users.ldif.j2 [hunk 3]
```diff
@@ -13,7 +13,7 @@ givenName: Joe
 mail: {{ __ldap_userid }}1@{{ __ldap_user_mail_domain }}
 uid: {{ __ldap_userid }}1
 description: I am interested in Robotics; I play Hockey and Soccer
-userPassword: {{ __ldap_user_admin_password }}
+userPassword: {{ __ldap_user_password }}
```
Comment (dejo1307) [inline roles/third_party/ldap-install/templates/users.ldif.j2:16]: Sorry, I missed this one - those are also two different passwords - in theory they can be the same, but in general the shouldn't?

**Review by dejo1307 — CHANGES_REQUESTED** [review]

**Pull request merged**